### PR TITLE
Add IBus to available RSSI input types

### PR DESCRIPTION
**Pull request by fxbisto**
RSSI option for users of the Flysky i6-type transmitters flashed with custom firmware (such as https://github.com/benb0jangles/FlySky-i6-Mod-) or newer models with native support for sending signal strength indicators over assignable aux channels.

### CODE REVIEW

**Comment by tracernz** [comment]
test this please

**Comment by fxbisto** [comment]
Sorry about that, I pasted the updated section from my pc to the web editor and somehow it missed off the : and ; from the IBUS section. Entire project now successfully compiles and tested working on a Revolution FC.

**Comment by tracernz** [comment]
test this please

File: ground/gcs/src/plugins/config/inputchannelform.cpp [hunk 1]
```diff
@@ -210,6 +210,9 @@ void inputChannelForm::groupUpdated()
         case ManualControlSettings::RSSITYPE_TBSCROSSFIRE:
             count = 12;
             break;
+          case ManualControlSettings::RSSITYPE_IBUS:
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/inputchannelform.cpp]: Please remove 2 spaces here so it lines up with the others. Otherwise, LGTM.

Comment (fxbisto) [reply ground/gcs/src/plugins/config/inputchannelform.cpp]: Sorry I missed this

**Comment by glowtape** [comment]
Wrong branch/PR, heh.

**Comment by fxbisto** [comment]
How did sanitycheck.c end up in here...was that added by my account ??

**Comment by tracernz** [comment]
Yeah, your PR is from your 'next' branch. I would recommend creating a new branch for each feature/PR you develop. We'll squash this when merging it so it ends up being just one commit.

**Comment by glowtape** [comment]
When Jenkins stops being baroque, I guess.

**Comment by fxbisto** [comment]
Got it. I'll be 100% sure each separate feature has it's own branch for the groups of commits from now on :)

**Comment by tracernz** [comment]
You managed to delete the file `flight/Libraries/sanitycheck.c` in https://github.com/d-ronin/dRonin/pull/2246/commits/931d31711c52b9bebd40221e6e9d39ce6b5eff8f

I removed the two misplaced commits from your next branch so we can merge this. I cherry-picked your sanity check change into another branch so you don't lose it: https://github.com/tracernz/dronin/commit/e9e26c874277bfba58c4c8d09ff4ed2092d55f63

**Comment by tracernz** [comment]
test this please

> When Jenkins stops being baroque, I guess.

It will only build if the user is whitelisted, otherwise it needs a manual kick each time. You don't want to go building random code from unknown people. ;)

**Comment by fxbisto** [comment]
Thankyou. I thought the delete command was to remove the commit from the pull request, not delete the entire file from the project.
I'll remember that for the future among other lessons learned today, thanks for your patience!

**Comment by tracernz** [comment]
No problem. git takes a while to learn. :)

**Pull request merged**